NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Dtb add post tag

# Description
> As an author I would like to be able to associate one or more Tags with one of my Posts so that readers can easily see the Post topics at a glance.
> 
> **Given** the author is viewing Post details
> **When** they select the `Manage Tags` option
> **Then** they should be presented with a selection of Tags options to associate with the Post
> 
> **Given** the author has selected the Tags they wish to associate with a Post
> **When** they click the `Save` button
> **Then** the association should be saved to the database
> **And** the user should be directed to the Post details page
> **And** the selected Tags should be displayed on the Post details page

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- pull down new branch and login
- navigate to Posts and select a Post which is not the current User's
- You should be routed to the details page of that Post. Ensure that there is no search bar in the top right of the Post Card
- navigate to MyPosts and select a Post
- From the details view, you should see a search bar in the top right.
- enter in a letter. A dropdown list of matches from the Tags database should appear with the first in the list being highlighted.
- Use the up and down arrows to highlight different tags.
- Click enter. The tag that was highlighted should appear in the search bar.
- Clear the search bar.
- enter in a letter. Click one of the options in the dropdown list.
- The tag that was clicked should appear in the search bar.
- Click the "+" button to add the tag. 
- You should see the tag below the Post's title.

# Checklist:
